src/components/PatientList/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { useTranslation } from 'react-i18next';
+import { PatientsList } from './styles';
+import PatientCard from '@components/PatientItem';
+import { LoadMoreButton } from '@components/general/styles';
+import { useGetPatientsQuery } from '../../services/PatientService';
+import { useState } from 'react';
+import { TWEETS_PER_LOAD, TWEETS_PER_PAGE } from '@constants/other';
+
+interface IProps {
+  searchValue: string;
+}
+
+function PatientList({ searchValue }: IProps) {
+  const { t } = useTranslation();
+  const { data: patients } = useGetPatientsQuery('');
+  const [currentPage] = useState<number>(1);
+  const [displayedTweets, setDisplayedTweets] =
+    useState<number>(TWEETS_PER_PAGE);
+  const startIndex = (currentPage - 1) * TWEETS_PER_PAGE;
```

Comment:
filter and pagination should be on backend. It is not rational to send a lot of patients to your client from the server. Instead you should send search params to server. But it of course depends on your backend)